dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgCountryAssociatedSuppliers.kt

```diff
@@ -0,0 +1,14 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * --- API model ---
+ * Fields of the LKSG questionnaire describing the number of suppliers a company has per country
+ */
+data class LksgCountryAssociatedSuppliers(
+    @field:JsonProperty(required = true)
+    val country: String,
```

Comment:
Will this be an ISO2 country code?   Seems like it in the draw.io.    So maybe `countryCode` ?

Reply:
Yes, but same here.

Reply:
In the current `LksgProcurementCategory`, we could call the field `numberOfSuppliersPerIso2CountryCode` or just
`numberOfSuppliersPerCountryCode` to reflect that country codes are used and no full names.

Reply:
Hmmm. I'm not sure if I agree. Sounds a bit like putting the data type into the objects name.

Reply:
Ok what about only adding `Code` to `numberOfSuppliersPerCountry`, to make it `numberOfSuppliersPerCountryCode` ?
I mean in our other models we also always use the term `countryCode` if it is an actual country code expected. (Of course it is just a string field in the end).

Just search in the IDE in the backend code for "ountry".   You'll see that we almost always use "countryCode" everywhere. Therefore, it would be consistent too. 

But if you are absolutetly against it, it is ok for me.

Reply:
Accepted. You're right. I will change that. However I still don't like the sound of it but can not put my finger on it.

Reply:
👍